### PR TITLE
[Author] [New] Add mdui@0.2.0 w/ npm auto-update via single package.json

### PR DESCRIPTION
**Pull request by zdhxiong**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/zdhxiong/mdui
 * Official website (optional, not the repository): http://mdui.org/
 * NPM package url (optional): https://www.npmjs.com/package/mdui
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 37
   - Count of stars: 1032
   - Count of forks: 93
 * NPM download stats (optional):
   - Downloads in the last day: 6
   - Downloads in the last week: 17
   - Downloads in the last month: 70

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/mdui/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "type": "git",
+    "url": "git+https://github.com/zdhxiong/mdui.git"
+  },
+  "homepage": "http://mdui.org",
+  "npmName": "mdui",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```
Comment (pvnr0082t) [inline ajax/libs/mdui/package.json]: @zdhxiong We should exclude `.txt` and `.ijmap` files. Let's use `"**/*.!(txt|ijmap)"`. Thanks.

**Comment by zdhxiong** [comment]
@pvnr0082t Please help review this PR again, thanks.

**Comment by PeterDaveHello** [comment]
@pvnr0082t I think you should help squash commits with commit message adjustment next time?

File: ajax/libs/mdui/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "mdui",
+  "filename": "mdui.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/mdui/package.json]: @pvnr0082t don't forget to check this.

**Pull request merged**